.EPDev/build-management/pipelines/tibco-streaming-samples/Jenkinsfile

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021-2023 Cloud Software Group, Inc.
+//
+//
+
+@Library('workflowLibs') _
+
+def jobUtil = new com.tibco.workflow.JobUtil()
+
+String platform = jobUtil.getJobPlatform(defaultPlatform: "linux")
+
+String executorLabel = jobUtil.getStrBuildLabelEx(
+    platform: platform, params: params,
+    base: [ all: "c.unix-tools",
+           linux: "(o.rh7 || o.almalinux7) && c.docker-compatible-linux-3.10 && c.unix-tools && q.str",
+           macos: "o.macos",
+           windows: "o.windows && a.x86_64" ],
+)
+
+pipeline {
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '2' ))
+        quietPeriod(600)
+        timeout(time: 4, unit: 'HOURS')
+        skipDefaultCheckout() // to subdirectory, see below
+    }
+
+    triggers {
+        // Poll once a day.
+        pollSCM('H 0 * * *’)
+    }
+
+    parameters {
+        choice(name: 'deployArtifacts', choices: ['never', 'default', 'always'], // never first; just for testing
+               description: 'The default is never to deploy because this is a test-only pipeline.')
+
+        string(name: 'mavenExtras', defaultValue: '',
+               description: 'Extra JVM options for Maven, usually -DskipTests and the like.')
+
+        booleanParam(name: 'cleanWorkspace', defaultValue: true, description: 'Clean workspace before build.')
+
```

Comment:
Add back the two standard email notification fields. Default for both should be just `change-authors`.